src/accordion/accordion.ts

```diff
@@ -199,6 +207,14 @@ export class NgbAccordion implements AfterContentChecked {
    */
   isOpen(panelId: string): boolean { return this._states.get(panelId); }
 
+  /**
+   * @internal
```

Comment:
> In any case let's try to in-line this method (and yes, you need to return null if you want to remove the attribute altogether).

The problem is that with inline `null` it becomes `false` and the attribute is not removed from DOM, it's added instead. That's the reason of a method. Personally I don't understand why result is different from methods vs inline.